metals/src/main/scala/scala/meta/internal/metals/MetalsLspService.scala

```diff
@@ -1150,6 +1155,39 @@ class MetalsLspService(
       .asJava
   }
 
+  private def maybeAmendScalaCliBspConfig(file: AbsolutePath): Future[Unit] = {
+    def isScalaCli = bspSession.exists(_.main.isScalaCLI)
+    def isScalaFile = file.toString.isScala || file.isAmmoniteScript
```

Comment:
```suggestion
    def isValidFile = file.toString.isScala || file.isJava || file.isAmmoniteScript
```
since Scala CLI can compile java files also?